src/org/zaproxy/zap/extension/ascanrulesBeta/SourceCodeDisclosureSVN.java

```diff
@@ -91,10 +95,7 @@ public String getName() {
 
 	@Override
 	public String getDescription() {
-		if (vuln != null) {
-			return vuln.getDescription();
-		}
-		return "Failed to load vulnerability description from file";
+		return Constant.messages.getString("ascanbeta.sourcecodedisclosure.desc");
```

Comment:
I know I should probably add a SVN specific one but I was trying to stay closest to the original code as possible. See https://github.com/zaproxy/zap-extensions/blob/beta/src/org/zaproxy/zap/extension/ascanrulesBeta/SourceCodeDisclosureSVN.java#L280